bcos-pbft/bcos-pbft/pbft/engine/PBFTEngine.cpp

```diff
@@ -775,6 +776,55 @@ bool PBFTEngine::checkProposalSignature(
         nodeInfo->nodeID(), _proposal->hash(), _proposal->signature());
 }
 
+bool PBFTEngine::checkRotateTransactionValid(
```

Comment:
should we do this in **RPBFTEngine**?

Reply:
for convenience of dynamic change consensus type, this function will return directly when consensus type is pbft.